src/Explorer/Panes/DeleteDatabaseConfirmationPanel.tsx

```diff
@@ -0,0 +1,187 @@
+import { useBoolean } from "@uifabric/react-hooks";
+import { Text, TextField } from "office-ui-fabric-react";
+import React, { FunctionComponent, useState } from "react";
+import LoadingIndicator_3Squares from "../../../images/LoadingIndicator_3Squares.gif";
+import * as Constants from "../../Common/Constants";
```

Comment:
The only thing we are using in this file is `Constants.Area`. Instead of importing the entire file we should just import area: `import { Area } from "../../Common/Constants";`

Reply:
Updated changes